src/test/groovy/unit/org/pih/warehouse/api/ProductApiTest.groovy

```diff
@@ -0,0 +1,178 @@
+package unit.org.pih.warehouse.api
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+import groovy.transform.builder.Builder
+import io.restassured.RestAssured
+import io.restassured.builder.RequestSpecBuilder
+import io.restassured.http.ContentType
+import io.restassured.http.Cookie
+import io.restassured.path.json.JsonPath
+import io.restassured.specification.RequestSpecification
+import lombok.AllArgsConstructor
+import lombok.Data
+import lombok.NoArgsConstructor
+import org.grails.web.json.JSONObject
+import spock.lang.Shared
+import spock.lang.Specification
+import static io.restassured.RestAssured.*
+import io.restassured.response.Response
+import static org.hamcrest.MatcherAssert.assertThat
+import static org.hamcrest.Matchers.is
+import static org.hamcrest.Matchers.notNullValue
+import static org.hamcrest.Matchers.lessThanOrEqualTo
```

Comment:
I think `import static` should be placed below the "normal" imports. This is also the default behavior when we are adding new imports through Intellij. And also we shouldn't import things using `*`, because there can be a lot of unnecessary imported things (I think this change request was mentioned before, but I am not sure)
```suggestion
import io.restassured.response.Response

import static io.restassured.RestAssured.*
import static org.hamcrest.MatcherAssert.assertThat
import static org.hamcrest.Matchers.is
import static org.hamcrest.Matchers.notNullValue
import static org.hamcrest.Matchers.lessThanOrEqualTo
```

Reply:
Check style seems to suggest the opposite. 
https://checkstyle.sourceforge.io/checks/imports/importorder.html#:~:text=group%20of%20static%20imports%20is,and%20aren't%20separated%20internally

So let's find a definitive source  or just ignore it.

Reply:
Hm, I can see similar imports to this which I suggested in our code
![obraz](https://github.com/openboxes/openboxes/assets/83239466/3455ac5a-5843-4618-add7-022488cce617)
If the static imports should be placed before normal imports (as I can see on the page you linked), then I am sorry for the confusion, I mean to keep those imports cleaner.
![obraz](https://github.com/openboxes/openboxes/assets/83239466/ff58eff3-8eb2-4b46-ac82-0837fc08425a)

Reply:
@alannadolny I honestly don't know what's best. When I saw the recommendation for putting static imports up top I was surprised. Just adding this to "we don't yet know the definitive answer" so we'll need to research it. I just don't want to spend too much time researching some of these syntax issues when we could be spending that time actually making tests work.